docs/resources/ui-building-blocks/pages/pages-configuration.md

```diff
@@ -20,15 +20,15 @@ This section allows you to define and manage parameters that your page can
 receive from other pages of the app. Parameters are essentially variables that
 hold values which can be passed between pages. For example, you might pass a
 user ID from a list page to a detail page to display specific information about
-that user.
+that user. Learn more about passing data between pages [here](../../../ff-concepts/navigation-routing/passing-data).
```

Comment:
Should be a admonition
:::info[LEARN MORE]

Reply:
Done!